spec/core/encoding.md

```diff
@@ -65,6 +65,31 @@ The address is the first 20-bytes of the SHA256 hash of the raw 32-byte public k
 address = SHA256(pubkey)[:20]
 ```
 
+The public key comprised of 32 bytes for one field element (the x-coordinate),
+plus one byte for the parity of the y-coordinate. The first byte depends is a
+0x02 byte if the y-coordinate is the lexicographically largest of the two
+associated with the x-coordinate. Otherwise the first byte is a 0x03. This
+prefix is followed with the x-coordinate.
+
+Decoding the RPC response for a secp256k1 pubkey:
+
+```sh
+# "pub_key": {
+#     "type": "tendermint/PubKeySecp256k1",
+#     "value": "AkeI23hsiCXVTf2+k+hGJAj/tuXRlwNRHI/Iv2Cvj3LQ"
+# },
+
+$ echo AkeI23hsiCXVTf2+k+hGJAj/tuXRlwNRHI/Iv2Cvj3LQ | base64 -D | xxd -p -c 999
```

Comment:
Note that `base64 -D` is for macOS, on linux it is `base64 -d`

Also, it might make sense to change `-c 999` to `-c 33` to be precise :)

```shell
$ echo AkeI23hsiCXVTf2+k+hGJAj/tuXRlwNRHI/Iv2Cvj3LQ | base64 -d | xxd -p -c 33
024788db786c8825d54dfdbe93e8462408ffb6e5d19703511c8fc8bf60af8f72d0
```